releasenotes/notes/add-release-notes-generation.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: release-notes/v1
+
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# kind describes the type of change that this represents. 
+# Valid Values are:
+# - bug-fix -- Used to specify that this change represents a bug fix. 
+# - security-fix -- Used to specify that this change represents a security fix. 
+# - feature -- Used to specify a new feature that has been added. 
+# - test -- Used to describe additional testing added. This file is optional for
+#   tests, but included for completeness.
+kind: feature
+
+# area describes the area that this change affects. 
+# Valid values are:
+# - traffic-management
+# - security
+# - telemetry
+# - installation
+# - istioctl
+# - documentation
+area: documentation
+
+# issue is a (potentially comma separated list) of GitHub issues resolved in this note.
```

Comment:
Wondering if this can be an array instead of a comma-separated list?

Reply:
Array might be better and more YAML like. I can update the template. I also want to add a title for upgrade notes.

Reply:
I think it's ok to do the number. The original challenge I was trying to solve with the URL was issues coming from multiple sources. If an Issue comes from istio/api, then providing just a number may point to the wrong issue. Same with test-infra and other repos. If just a number is specified, we can assume the issue is from istio/istio. If a URL is specified, we can use that instead.